apps/sistema-demo/src/app/app.component.ts

```diff
@@ -56,10 +56,11 @@ import {
   SistemaLabelComponent,
   SistemaPasswordInputComponent,
   SistemaSpinnerComponent,
+  SistemaStickyFooterComponent,
+  SistemaTwoColumnsConfigurationItemComponent,
 } from './sistema-pages';
 import { SliderPageComponent } from '../../../../libs/pastanaga-angular/projects/demo/src/app/demo/pages/slider-page/slider-page.component';
```

Comment:
wrong import

Reply:
still not, this is how we do the imports of pastanaga document pages in sistema-demo (documentation is in the demo project from pastanaga, which is not exported)